go.mod

```diff
@@ -4,6 +4,7 @@ require (
 	github.com/brancz/gojsontoyaml v0.0.0-20190425155809-e8bd32d46b3d
 	github.com/campoy/embedmd v1.0.0
 	github.com/dgryski/go-jump v0.0.0-20170409065014-e1f439676b57
+	github.com/gogo/protobuf v1.1.1 // indirect
```

Comment:
Do we need this? We need to make sure we are running `go mod tidy` with the latest go version.